inspirehep/utils/normalizers.py

```diff
@@ -20,16 +20,36 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-"""Models related to INSPIRE depositions."""
-
 from __future__ import absolute_import, division, print_function
 
-from inspirehep.dojson.model import FilterOverdo, add_schema, clean_record
+from sqlalchemy.orm.exc import NoResultFound
+from invenio_oauthclient.models import UserIdentity
+
+from inspirehep.modules.search.api import JournalsSearch
+
+
+def normalized_title(journal_title):
+    normalized_journal_title = journal_title
+    hits = JournalsSearch().query(
+        'match',
+        title_variants__title__lowercased=journal_title
+    ).execute()
+
+    if hits:
+        try:
+            normalized_journal_title = hits[0].short_titles[0].title
+        except (AttributeError, IndexError):
+            pass
+    return normalized_journal_title
 
 
-filters = [
-    add_schema('hep.json'),
-    clean_record,
-]
+def normalized_orcid(id_user):
```

Comment:
I don't see any normalization happening here, just retrieving from DB. Is this function used from several places or you can do the query directly in the place where you need it.